lms/static/sass/course/_info.scss

```diff
@@ -69,6 +69,19 @@ div.info-wrapper {
       margin-bottom: lh();
       padding-left: 0;
 
+      .show-older-updates {
+        -webkit-border-radius: 5px;
+        -moz-border-radius: 5px;
+        border-radius: 5px;
+        border: solid 2px #74B8D8;
+        width: 100%;
+        display: block;
+        text-align: center;
+        padding: 10px 0;
+        font-size: 14px;
```

Comment:
We should have appropriate extends in `lms/static/sass/elements/_typography.scss`.